docs/README.md

```diff
@@ -92,7 +92,27 @@ See [config.md#instrumented-libraries-and-frameworks](config.md#instrumented-lib
 Download and extract the appropriate binaries from
 [the latest release](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest).
 
-> The path where you put the binaries is referenced as `%InstallationLocation%`
+> The path where you put the binaries is referenced as `$INSTALL_DIR`
+
+You can use a convenient Bash script to do it.
+Parameters:
+
+- `INSTALL_DIR` - location where binaries are to be installed;
+  default: `./otel-dotnet-auto`
+- `FLAVOUR` - possible values: `linux-glibc`, `linux-musl`, `macos`, `windows`;
+  required
+- `RELEASES_URL` - GitHub releases URL;
+  default: `https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases`
+- `TMPDIR` - temporary directory used when downloading the files;
+  default: `$(mktemp -d)`  
+- `VERSION` - version to download;
+  required
+
+```sh
+( set -o pipefail
+curl -sSfL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/main/download.sh |
+  VERSION=v0.3.1-beta.1 FLAVOUR=linux-glibc bash -s )
```

Comment:
Extend release process to update this line.

Reply:
PTAL 83e999a3b